core/templates/dev/head/pages/splash.html

```diff
@@ -22,101 +22,101 @@
     <div class="oppia-splash-section-one text-center">
       <h1 class="oppia-splash-h1" translate="I18N_SPLASH_TITLE"></h1>
       <div class="oppia-splash-background-icon-row">
-        <img src="/images/splash/books.svg" class="oppia-splash-books">
-
-        <img src="/images/subjects/Humor.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Combinatorics.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Cooking.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Government.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Architecture.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/History.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Microbiology.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Engineering.svg" class="oppia-splash-background-icon">
-
-        <img src="/images/subjects/Algorithms.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Economics.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Computing.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Reading.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Art.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Creativity.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Physics.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Language.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Arithmetic.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Chess.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Astronomy.svg" class="oppia-splash-background-icon">
-
-        <img src="/images/subjects/Religion.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Mathematics.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Philosophy.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Humor.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Combinatorics.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Cooking.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Government.svg" class="oppia-splash-background-icon">
-        <img src="/images/subjects/Architecture.svg" class="oppia-splash-background-icon">
+        <img ng-src="<[getStaticImageUrl('/splash/books.svg')]>" class="oppia-splash-books">
+
+        <img ng-src="<[getSubjectsImageUrl('Humor')]>" class="oppia-splash-background-icon">
```

Comment:
Single subject, so getSubjectImageUrl -- or, for parity, probably better to use getStaticSubjectImageUrl